packages/cli-doctor/src/tools/checkInstallation.ts

```diff
@@ -19,11 +19,13 @@ const isSoftwareNotInstalled = async (command: string): Promise<boolean> => {
 const doesSoftwareNeedToBeFixed = ({
   version,
   versionRange,
+  looseRange = false,
```

Comment:
default value so no other code has to change, and follows the semver package historical precedent of all semver APIs taking loose as a final argument, with false being the default